.circleci/config.yml

```diff
@@ -13,6 +15,13 @@ jobs:
           key: v1-dependency-cache-{{ checksum "st2/requirements.txt" }}
       - run: sudo apt install python-dev
       - run: make docs
+      - run:
+          name: Store HTML docs in workspace dir
+          command: mkdir /tmp/st2docs && cp -r docs/build/html/* /tmp/st2docs/
+      - persist_to_workspace:
```

Comment:
Does it also makes sense to copy those dirs to CircleCI artifacts?
May help with some future debugging.

Reply:
I wondered about that. Might make sense.

Reply:
Done - now storing artifacts. Works OK. 
https://github.com/StackStorm/st2docs/pull/849/commits/5aa1246438fff60a9fbdde27342b16ce34a519d8